core/modules/qproc/testQueryAnaGeneral.cc

```diff
@@ -1067,7 +1067,7 @@ BOOST_AUTO_TEST_CASE(Case01_2006) {
     std::string stmt = "SELECT scisql_fluxToAbMag(uFlux_PS) "
         "FROM   Object WHERE  (objectId % 100 ) = 40;";
     // % is not a valid arithmetic operator in SQL92.
-    char const expectedErr[] = "ParseException:Error parsing query, near \"%\"";
+    char const expectedErr[] = "ParseException:Error parsing query, near \"%\", Unhandled operator type:%";
```

Comment:
I understand this code was there for year. However, it looks a bit dangerous to depend on a very specific error message.

Reply:
(same)